hls4ml/model/layers.py

```diff
@@ -850,6 +850,7 @@ class SimpleRNN(Layer):
         Attribute('return_sequences', value_type=bool, default=False),
         Attribute('return_state', value_type=bool, default=False),
         ChoiceAttribute('direction', ['forward', 'backward'], default='forward'),
+        Attribute('static', value_type=bool, default=True),
```

Comment:
This is an implementation detail, so it should belong to the backend. [Here](https://github.com/fastmachinelearning/hls4ml/blob/master/hls4ml/backends/vivado/vivado_backend.py#L25).

Reply:
Ok, I've moved it there.